composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^7.1.3",
         "symfony/process": "^4.2",
         "phpstan/phpstan": "^0.11.1",
-        "orchestra/testbench": "^3.6",
+        "orchestra/testbench": "^3.8",
```

Comment:
It's a bit odd that you supporting Laravel 5.6-5.8 but only loads `testbench` `3.8` for testing (which only going to tests against Laravel 5.8).

Reply:
I found this _issue_ in other packages as well. How would one go about automating the tests to check a package against different versions of a requirement (like travis' php versions array)?

As you say it only gets the latest version and tests against that, so _support_ for other versions is not really tested, it's just well intended.

Reply:
https://travis-ci.org/laravie/authen/builds/498431921

It's not very hard to enable support for multiple versions, see https://github.com/laravie/authen/blob/master/.travis.yml for example.

Reply:
Awesome! Thanks @crynobone, I'll need to check the Travis config file more in detail.